packages/pf3-component-mapper/src/form-fields/button.js

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
-import { Button } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import { Button as PFButton } from 'patternfly-react';
 
-const ButtonOverride = (props) => <Button style={{ marginLeft: 3 }} { ...props } />;
+export const Button = ({ label, variant, className, ...rest }) => {
+  const { formOptions, FieldProvider, validate, ...props } = { ...rest };
```

Comment:
You can do this in the arguments. No need for another variable in memory.

Reply:
Linter does not like it if I do:
```js
export const Button = ({ label, variant, className, formOptions, FieldProvider, validate, ...props }) => {
  ...
};

```

Reply:
Oh well, let's keep it then.